<?xml version="1.0"?>

<launch>

    <arg name="chatter" default="chatter" />
    <!--arg name="laser_out" default="/hsrb/head_rgbd_sensor/depth_registered/laser_scan" /-->    
    

    <!-- run listener and talker nodes -->
    <node pkg="beginner_tutorials" type="talker" name="talker" output="screen">

        <remap from="chatter" to="$(arg chatter)"/>
    </node>  
      

   
</launch>
